.github/workflows/e2e.yml

```diff
@@ -3,192 +3,296 @@ name: "e2e"
             "registry": "fluence"
           },
           {
             "package": "avm-server",
-            "version": "=${{ needs.aquavm.outputs.cargo-version }}",
+            "version": "=${{ needs.aquavm-crates.outputs.version }}",
             "registry": "fluence"
           },
           {
             "package": "air-interpreter-wasm",
-            "version": "=${{ needs.aquavm.outputs.cargo-version }}",
+            "version": "=${{ needs.aquavm-crates.outputs.version }}",
+            "registry": "fluence"
+          },
+          {
+            "package": "fluence-spell-dtos",
+            "version": "=${{ needs.spell.outputs.cargo-version }}",
+            "registry": "fluence"
+          },
+          {
+            "package": "fluence-spell-distro",
+            "version": "=${{ needs.spell.outputs.cargo-version }}",
             "registry": "fluence"
           }
         ]
 
   js-client-tests:
     name: "js-client"
     needs:
-      - marine
-      - aquavm
-      - rust-peer
+      - marine-js
+      - aquavm-avm
+      - rust-peer-snapshot
 
     uses: fluencelabs/js-client/.github/workflows/tests.yml@master
     with:
-      marine-js-version: "${{ needs.marine.outputs.marine-js-version }}"
-      avm-version: "${{ needs.aquavm.outputs.avm-version }}"
-      rust-peer-image: "${{ needs.rust-peer.outputs.rust-peer-image }}"
+      marine-js-version: "${{ needs.marine-js.outputs.version }}"
+      avm-version: "${{ needs.aquavm-avm.outputs.version }}"
+      rust-peer-image: "${{ needs.rust-peer-snapshot.outputs.rust-peer-image }}"
 
   js-client:
     needs:
-      - marine
-      - aquavm
+      - marine-js
+      - aquavm-avm
 
     uses: fluencelabs/js-client/.github/workflows/snapshot.yml@master
     with:
-      marine-js-version: "${{ needs.marine.outputs.marine-js-version }}"
-      avm-version: "${{ needs.aquavm.outputs.avm-version }}"
+      marine-js-version: "${{ needs.marine-js.outputs.version }}"
+      avm-version: "${{ needs.aquavm-avm.outputs.version }}"
+
+  fluence-cli-snapshot:
+    name: "fluence-cli"
+    needs:
+      - js-client
+
+    uses: fluencelabs/fluence-cli/.github/workflows/snapshot.yml@main
+    with:
+      js-client-snapshots: "${{ needs.js-client.outputs.js-client-snapshots }}"
 
   aqua-playground:
     needs:
       - js-client
-      - rust-peer
+      - rust-peer-snapshot
 
     uses: fluencelabs/aqua-playground/.github/workflows/tests.yml@master
     with:
       js-client-snapshots: "${{ needs.js-client.outputs.js-client-snapshots }}"
-      rust-peer-image: "${{ needs.rust-peer.outputs.rust-peer-image }}"
+      rust-peer-image: "${{ needs.rust-peer-snapshot.outputs.rust-peer-image }}"
+
+  # spell-aqua-tests:
+  #   name: "spell"
+  #   needs:
+  #     - rust-peer-snapshot
+  #   uses: fluencelabs/spell/.github/workflows/tests.yml@main
+  #   with:
+  #     rust-peer-image: "${{ needs.rust-peer-snapshot.outputs.rust-peer-image }}"
```

Comment:
lets remove or enable it?

Reply:
I will be fixed soon and I will uncomment it

Reply:
Then comment the commented code with this message 👍